Olympus.Stratis/functions/fn_environmentControl.sqf

```diff
@@ -25,23 +25,28 @@ private _timeAction = [
     {
         private _timeSettings = [
             // [time, "Display Name"]
-            [1, "0100"],
-            [3, "0300"],
-            [5, "0500"],
-            [7, "0700"],
-            [9, "0900"],
-            [11, "1100"],
-            [13, "1300"],
-            [15, "1500"],
-            [17, "1700"],
-            [19, "1900"],
-            [21, "2100"],
-            [23, "2300"]
+            [1],
```

Comment:
You don't need an array in an array anymore. :D

Actually you don't need to define that at all. Just do:
```sqf
for "_time" from 1 to 23 step 2 do {
    //_time
};
```